pages.nl/common/inkscape.md

```diff
@@ -0,0 +1,33 @@
+# inkscape
+
+> Een SVG (Scalable Vector Graphics) bewerkingsprogramma.
+> Voor Inkscape versies tot en met 0.92.x, gebruik -e in plaats van -o.
+> Meer informatie: <https://inkscape.org>.
+
+- Open een SVG-bestand in de Inkscape GUI:
+
+`inkscape {{bestandsnaam.svg}}`
+
+- Exporteer een SVG-bestand in een bitmap met het standaardformaat (PNG) en de standaardresolutie (96 DPI):
+
+`inkscape {{bestandsnaam.svg}} -o {{bestandsnaam.png}}`
+
+- Exporteer een SVG-bestand in een bitmap van 600x400 pixels (vervorming van de aspectverhouding mogelijk):
+
+`inkscape {{bestandsnaam.svg}} -o {{bestandsnaam.png}} -w {{600}} -h {{400}}`
+
+- Exporteer de tekening (selectiekader van alle objecten) van een SVG-bestand in een bitmap:
+
+`inkscape {{bestandsnaam.svg}} -o {{bestandsnaam.png}} -D`
+
+- Exporteer een enkel object, gezien zijn ID, in een bitmap:
+
+`inkscape {{bestandsnaam.svg}} -i {{id}} -o {{object.png}}`
+
+- Exporteer een SVG-document naar PDF, converteer alle teksten naar paden:
+
+`inkscape {{bestandsnaam.svg}} -o {{bestandsnaam.pdf}} --export-text-to-path`
+
+- Dupliceer het object met id="pad123", roteer het duplicaat 90 graden, sla het bestand op, en sluit Inkscape af:
+
+`inkscape {{bestandsnaam.svg}} --select=path123 --verb="EditDuplicate;ObjectRotate90;FileSave;FileQuit"`
```

Comment:
The argument options should be between `{{}}`, the same fix applies to the English page:
```suggestion
`inkscape {{bestandsnaam.svg}} --select=path123 --verb="{{EditDuplicate;ObjectRotate90;FileSave;FileQuit}}"`
```

Reply:
I haven't used the Inkscape CLI myself, but it seems to me that this is part of the command as described above. Not a parameter template, or option?

If this was describing generically how to use the `--verb` argument to trigger actions in Inkscape, I would agree. But this section is specifically describing how to duplicate an object (`EditDuplicate`), rotate the object (`ObjectRotate90`), save the file (`SaveFile`), and quit Inkscape (`FileQuit`).

Please let me know if my understanding is incorrect.

Reply:
@JenniX3 No, just check the last example in [tokei](https://github.com/tldr-pages/tldr/blob/master/pages/common/tokei.md). That's what I mean.

Reply:
Ahh, understood. Sure, I'll commit the suggestion then.
I'll also submit a PR for the English as well later.